disco_aws_automation/disco_datapipeline.py

```diff
@@ -25,6 +25,8 @@ class DataPipelineConsts(object):
     TEMPLATE_DIR = "datapipeline_templates"
     LOG_LOCATION_FIELD = "pipelineLogUri"
     SUBNET_ID_FIELD = 'subnetId'
+    BACKUP_PERIOD = 'period'
+    DAILY_SCHEDULE = 'DailySchedule'
```

Comment:
@aoskotsky-amplify this value is used to get schedule object from pipeline objects and then to modify it.